src/modules/launcher/PowerLauncher/ViewModel/MainViewModel.cs

```diff
@@ -1016,27 +1016,30 @@ public void ToggleWox()
                 var window = Application.Current.MainWindow;
                 Wpf.Ui.Controls.WindowBackdrop.RemoveBackground(window);
 
-                // Taken from WPFUI's fix for the title bar issue. We should be able to remove this fix when WPF UI 4 is integrated.
-                // https://github.com/lepoco/wpfui/pull/1122/files#diff-196b404f4db09632665ef546da6c8e57302b2f3e3d082eb4b5c295ae3482d94a
-                IntPtr windowHandle = new WindowInteropHelper(window).Handle;
-                if (windowHandle == IntPtr.Zero)
+                if (OSVersionHelper.IsWindows11())
```

Comment:
Do we need a comment why this is only on Win11?

Reply:
I can add it :)

Reply:
I've added a comment.